src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -177,7 +178,16 @@ def open_browser(self, url=None, browser='firefox', alias=None,
         `add_argument ( "--headless" )` is same as
         `add_argument("--headless")`. But `add_argument(" --headless ")` is
         not same same as `add_argument ( "--headless" )`, because
-        spaces inside of quotes are not removed.
+        spaces inside of quotes are not removed. Please note that if
+        options string contains backslash, example a Windows OS path,
+        the backslash needs escaping both in Robot Framework data and
+        in Python side. This means single backslash must be writen using
+        four backslash characters. Example, Windows path:
+        "C:\\path\\to\\profile" must be written as
+        "C:\\\\\\\\path\\\\\\to\\\\\\\\profile". Another way to write
```

Comment:
`\\\\\\\\` looks somewhat strange here in source code but is probably right. Prefixing this docstring with `r` would allow using `\\\\` instead.

Reply:
Could also mention that most of the time `/` works as a path separator also on Windows.

Reply:
Good idea, I will try it out.

Reply:
For the first comment, it looks really strange, but I generated kw doc's and it did render correctly.